Frontend/src/pages/detail/detail.module.scss

```diff
@@ -58,7 +58,10 @@
   .fullname {
     @include headerTitle;
     text-transform: capitalize;
-  }
+	}
+	.fulnameDeath {
```

Comment:
uz parkrat jsem si vsiml, ze se Ti v cssku michaji mezery a taby?

Reply:
Mely by byt jen mezery. Az dodelam nejakou automatiku formatovani pred commitem, tak se to uz stavat nebude.